<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTBAlarmConfigurationWidget</class>
 <widget class="QWidget" name="QTBAlarmConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="connectedCheckBox">
       <property name="text">
        <string>Connected</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Configuration name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="configNameLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="alarmsTableWidget">
       <property name="minimumSize">
        <size>
         <width>700</width>
         <height>400</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Active</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Watched Parameter</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Function</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Function Arg Value</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Message</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Displayed Parameter Value</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Color</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="upPushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/icons/icons.qrc">
         <normaloff>:/icons8_up4_16px.png</normaloff>:/icons8_up4_16px.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downPushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/icons/icons.qrc">
         <normaloff>:/icons8_down4_16px.png</normaloff>:/icons8_down4_16px.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="newPushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/icons/icons.qrc">
         <normaloff>:/icons8_plus_math_32px_1.png</normaloff>:/icons8_plus_math_32px_1.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="delPushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/icons/icons.qrc">
         <normaloff>:/icons8_minus_math_32px.png</normaloff>:/icons8_minus_math_32px.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
